openfisca_us/variables/hhs/medicaid.py

```diff
@@ -0,0 +1,37 @@
+from openfisca_core.model_api import *
```

Comment:
You can now replace all the imports with this
```suggestion
from openfisca_us.model_api import *
```